src/components/AssociationList.tsx

```diff
@@ -0,0 +1,31 @@
+import { useAssociations } from "@/contexts/AssociationsContext";
+import { Card, CardDescription, CardImage, CardTitle } from "./ui/card";
+import { Button } from "./ui/button";
+
+export default function AssociationList() {
+  const { data, isPending, error } = useAssociations();
+
+  return (
+    <div className="flex flex-col">
+      {isPending && <p>Loading associations...</p>}
+      {error && <p>Error fetching associations!</p>}
+
+      {data?.length === 0 && <p>No associations found.</p>}
+
+      {data?.map((association) => (
+        <Card key={association.id}>
+          <CardImage src={association.logo} alt={`${association.name} logo`} />{" "}
+          <div className="flex flex-col p-4 leading-normal">
+            <div className="flex flex-row justify-between leading-normal">
+              <CardTitle>{association.name}</CardTitle>
+              <Button className="bg-shadowDark">Learn more</Button>
```

Comment:
use variants to change the button
maybe you wanted to use "icon"
if "icon" is not clear as name, maybe we can change it as "status" or something like that, but still check if it is more helpful since it has also some "on hover" functionalities and etc.